scripts/pr-convergence.py

```
@@ -0,0 +1,971 @@
+                return False
+        if arg.startswith("--method=") and arg.split("=", 1)[1].upper() != "GET":
+            return False
+    return True
+
+
+class GhClient:
+    def __init__(self, repo: str) -> None:
+        self.repo = repo
+        self.owner, self.name = parse_repo(repo)
+
+    def run(self, args: list[str]) -> Any:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `run`

(ANN401)